example.env

```diff
@@ -22,10 +22,12 @@ GATEKEEPER_FARM_CALENDAR_API=http://farmcalendar:8002/api/v1/
 GATEKEEPER_FARM_CALENDAR_POST_AUTH=http://localhost:8002/post_auth/
 
 GATEKEEPER_LOGIN_URL=http://localhost:8001/login/
+GATEKEEPER_LOGIN_API_URL=http://localhost:8001/api/token/
```

Comment:
@pranavnbapat is this the endpoint for checking the token? if so than I believe it's best to change this env var to something like GATEKEEPER_TOKEN_VERIFY_URL or something like that.